legal/ADDITIONAL_LICENSE_INFO

```diff
@@ -1 +1 @@
-../java.base/ADDITIONAL_LICENSE_INFO
\ No newline at end of file
+Please see ..\java.base\ADDITIONAL_LICENSE_INFO
```

Comment:
This was added/ changed by IntelliJ's JavaDoc tool.. Keep or undo?

Reply:
It looks like it depends on OS (Linux vs Windows) and JDK version/flavour. I'd say do as you wish, it doesn't really matter. Same for similar changes.